protocol/x/subaccounts/keeper/subaccount.go

```diff
@@ -609,6 +609,17 @@ func (k Keeper) internalCanUpdateSubaccounts(
 		}
 	}
 
+	var perpOpenInterestDelta *perptypes.OpenInterestDelta
+	// If update type is `Match`, calculate the open interest delta for the updated perpetual.
+	// Note: this assumes that `Match` can only happen for perpetuals.
```

Comment:
AFAIK this assumption will hold true for a very long time (we don't have plans to support spot), which is why I didn't include a TODO.